### PR TITLE
Check that the `install` target does something vaguely sensible

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

**Comment by ltratt** [comment]
bors try

**Comment by ltratt** [comment]
bors try

**Comment by ltratt** [comment]
bors try

**Comment by ltratt** [comment]
bors try

**Comment by ltratt** [comment]
bors

**Comment by ltratt** [comment]
bors try

**Comment by ltratt** [comment]
OK, that was a pallaver. @vext01 This will need some serious squashing before merging!

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,7 @@ install:
 	install -c -m 444 snare.conf.example ${PREFIX}/share/examples/snare
 
 distrib:
+	git status
```
Comment (vext01) [inline Makefile]: Is this for the logs?

Comment (ltratt) [reply Makefile]: Mea culpa -- I should have removed that! Fixed in https://github.com/softdevteam/snare/pull/38/commits/7c8cd889688b3d6e3d2bdf667cd8bb9cdbe74e32.

**Comment by vext01** [comment]
Please squash

**Comment by ltratt** [comment]
Squashed (and PR title changed accordingly).

**Pull request merged**